injector/Injectors/InlineInjector.py

```
@@ -0,0 +1,66 @@
+import ast
+from injector.VariableCollectors.CollectVariableDefault import CollectVariableDefault
+from injector.helper import getEmptyRootNode
+
+from injector.helper import getVarLogStmt, getLtLogStmt, getUniqueIdNamedAssignStmt, getLocalVarsLogStmt
```

Ruff (0.11.9)

`injector.helper.getVarLogStmt` imported but unused

Remove unused import: `injector.helper.getVarLogStmt`

(F401)